conf/distro/include/thunder_generic.conf

```diff
@@ -2,7 +2,7 @@
 DISTRO_FEATURES_append = " compositor"
 # Enable thunder flag to enable thunder specific events
 DISTRO_FEATURES_append = " thunder"
-WPE_THUNDER_BUILDTYPE = "thunder_release"
+WPE_THUNDER_BUILDTYPE = "thunder_debugoptimized"
```

Comment:
Shouldn't this be set in the debug config instead of generic? By default the frameowrk should build in the release version

Reply:
oh..this was wrong commit, will remove